cnames_active.js

```diff
@@ -2974,6 +2974,7 @@ var cnames_active = {
   "wdd": "wangduanduan.github.io",
   "weather": "phanendraguptha.github.io/weather",
   "weaver": "maxkfranz.github.io/weaver", // noCF? (don´t add this in a new PR)
+  "web.quizlet": "redyetidev.github.io/Quizlet.JS-Web",
```

Comment:
```suggestion
  "web.quizlet": "redyetidev.github.io/Quizlet.JS-Web", // noCF
```
As this is a sub-sub-domain, the Cloudflare proxy cannot be enabled